proto/xudrpc.proto

```diff
@@ -191,10 +191,12 @@ message Order {
   string host_id = 5 [json_name = "host_id"];
   // A UUID for this order
   string id = 6 [json_name = "id"];
+  // The local ID for this oder
+  string local_id = 7 [json_name = "local_id"];
```

Comment:
`GetOrders` and `SubscribePeerOrders` show `PeerOrder` with the field `id`. So we can't remove `id` from this message.

Reply:
@michael1011 is right in that we're we're dropping `localId` when broadcasting an order, so a `peerOrder` doesn't have that field. 
@sangaman is right in that `id` is irrelevant for an `ownOrder`, because it suppose to be invisible to the client.
 
At the moment `Order` message definition contains all possible fields. Many calls using it, but require only some of the fields. I think we should define all the types more precisely. 
If i'm creating a new `PlaceOrderRequest` object, I might end up thinking that I need to set `HostId` \ `Invoice` \ `CreatedAt`,  which in practice would be ignored. 
If you agree on this, it can be addressed in a separate PR.

Reply:
Yes, `PlaceOrderRequest` ought to be fixed to accept the same parameters that are possible on the xucli placeorder call. Doesn't have to be part of this PR but that's something that could be improved.

I originally thought we can drop id even though `GetOrders` and `SubscribePeerOrders` use peer orders because I suspected there's no value in returning the xud global order id to the client. But in thinking about this some more, we will need it for the use case where the exchange does all the matching - they need to be able to specify which peer orders to take by the XUD id.

However, I'm also thinking now we might have a case where an exchange is letting XUD do all the matching and doesn't want to have to create and track their own Ids. Which makes me think maybe `localId` should be optional when placing an order. Maybe even tracking localIds at all can be something that's configurable? Also something that can be done in a separate PR, but I wanted to see what you thought.

Reply:
I agree that there are cases in which tracking `localId` is not necessary. 

Can I take care of that and separating the gRPC messages? Because when this PR gets merged all of my issues for the prealpha are done.

Reply:
@kilrau what do you say about `localId` being optional?

If so, yes, it can and better be done in this PR. gRPC messages separation is not important for pre-alpha IMO.